poem/src/middleware/tracing_mw.rs

```diff
@@ -47,7 +47,7 @@ impl<E: Endpoint> Endpoint for TracingEndpoint<E> {
         );
 
         if let Some(path_pattern) = req.data::<PathPattern>() {
-            span.record("path_pattern", path_pattern.0.as_ref());
+            span.record("path_pattern", &path_pattern.0.as_ref());
```

Comment:
@sunli829 This was necessary to make poem compile locally. Not sure why it passed in CI/CD in the old version.

I compiled with rust 1.65